### ISSUE TITLE
Boundary conditions for `draw.ellipse_perimeter`

### ISSUE THREAD

**Issue by Borda** [scikit-image/scikit-image#2480]
## Description
I am wondering if the circular drawing of an ellipse is proper behavior, such that overflow section of an ellipse on the left side appears on the right side of the image...

## Way to reproduce
- [x] Code example
```
import numpy as np
from skimage.draw import ellipse_perimeter

image = np.zeros((9, 15), dtype=np.uint8)
rr, cc = ellipse_perimeter(4, 5, 3, 7)
image[rr, cc] = 1

import pylab
pylab.imshow(image, interpolation="nearest", cmap='gray')
pylab.show()
```
- [x] Relevant images (if any)
![figure_1](https://cloud.githubusercontent.com/assets/6035284/22429835/5307812a-e70c-11e6-9d1c-cce45d374dca.png)
- [x] scikit-image version '0.11.3'

**Comment by soupault** [comment]
In my opinion, a better default behaviour should be to truncate the values outside the image.
Maybe we can think of a parameterization here, but I personally cannot imagine any tasks in which the current behaviour is needed.

**Comment by sciunto** [comment]
+1

**Comment by stefanv** [comment]
The behavior wanted here can be generated with: `rr, cc = ellipse_perimeter(4, 5, 3, 7, shape=image.shape)`.

If you want to clip by default, you always have to specify the image dimensions, so I don't see that as a win.

**Comment by Borda** [comment]
ok, but I would at least to put such example of ellipse with and without `shape` parameter

**Comment by stefanv** [comment]
@Borda Yes, absolutely!  Would you be interesting in making a PR?

**Comment by Borda** [comment]
@stefanv have look at #2524

**Comment by soupault** [comment]
Closed via #2524 .
